sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala

```diff
@@ -67,7 +67,7 @@ class StateOperatorProgress private[sql](
  *                Similarly, when there is no data to be processed, the batchId will not be
  *                incremented.
  * @param durationMs The amount of time taken to perform various operations in milliseconds.
- * @param currentWatermark The current event time watermark in milliseconds
+ * @param eventTime Statistics of event time seen in this batch
```

Comment:
nit: since you are touching this file, could you also fix the comment of `@param timestamp`? It's better to document the format as well, such as, `The beginning time of the trigger in ISO8601 format. (e.g., 2016-12-05T20:54:20.827Z)`